### ISSUE TITLE
Consistent lineendings

### ISSUE THREAD

**Issue by codecreativity** [GamesResearchTUG/Maroon#69]
Some files use Windows line endings, some files use Unix line endings.
Select one and use it consistently?

**Comment by michaelholly** [comment]
You are right. I would suggest Unix-style. I think Windows users should set the following configuration "Checkout Windows-style, commit Unix-style line endings" during the git setup. Git will then convert LF to CRLF when checking out and CRLF will be converted to LF when committing.

**Comment by codecreativity** [comment]
Do *not* let git convert line-endings when checking out files, this tampers with 3rd party files that need to have a given sha256 hash. Always select "Checkout as-is"
